build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -27,7 +27,7 @@
     <!-- This dependency was added due to an issue in restore where a lower version of this package coming from nuget.commandline.xplat
     led to an error. This is tracked as NuGet issue : https://github.com/NuGet/Home/issues/4213 -->
     <PackageReference Include="Microsoft.Build.Framework" Version="$(CLI_MSBuild_Version)" />
-    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="1.0.3" />
```

Comment:
We are upgrading the cli to use 2.0 for platformabstractions but the bootstrapped cli depends on the 1.0.3

Once the new CLI comes out we can revert this change